coalib/bears/GlobalBear.py

```diff
@@ -4,22 +4,28 @@
 
 class GlobalBear(Bear):
     """
-    A GlobalBear is able to analyze semantic facts across several file.
+    A GlobalBear analyzes semantic facts across several files.
 
     The results of a GlobalBear will be presented grouped by the origin Bear.
-    Therefore Results spanning above multiple files are allowed and will be
-    handled right.
+    Therefore Results spanning across multiple files are allowed and will be
+    handled correctly.
 
-    If you only look at one file at once anyway a LocalBear is better for your
-    needs. (And better for performance and usability for both user and
-    developer.)
+    If you are inspecting a single file at a time, you should consider
+    using a LocalBear.
     """
 
     def __init__(self,
-                 file_dict,  # filename : file contents
+                 file_dict,
                  section,
                  message_queue,
                  timeout=0):
+        """
+        Constructs a new GlobalBear.
+
+        :param file_dict: The dictionary of {filename: file contents}.
+
```

Comment:
> When changing documentation, you have to track down all the duplicated places. Duplication is always bad, be it either code or docs.

imo, it's a usability v/s maintainability tradeoff, more documentation will help new developers and bears writers find and implement things easily.

> However, there is some "metaclass conflict" which is probably the root cause of that (I have no idea though). So, please either be more specific and/or let's hear a third opinion about duplication.

I've not heard of "metaclass conflict" before, so yeah, let's wait for third opinion :)